### PR TITLE
chore: make more showcases

### PR DESCRIPTION
**Pull request by craigbilner**
This rather small PR converts the rest of the stories to showcases

### CODE REVIEW

File: packages/author-profile/author-profile.showcase.js [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+const preventDefaultedAction = decorateAction =>
+  decorateAction([
+    ([e, ...args]) => {
+      e.preventDefault();
+      return ["[SyntheticEvent (storybook prevented default)]", ...args];
+    }
+  ]);
+
+const slug = "deborah-haynes";
+const pageSize = 5;
```
Comment (L0wry) [inline packages/author-profile/author-profile.showcase.js]: fairly sure that old requirements for page size were 10. Not sure if this matters

File: packages/error-view/error-view.showcase.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+import React, { Component } from "react";
+import { View, ViewPropTypes, Text, StyleSheet } from "react-native";
+import PropTypes from "prop-types";
+import Video from "@times-components/video";
+import { fontSizes } from "@times-components/styleguide";
+import ErrorView from "./src/error-view";
+
+const { style: ViewPropTypesStyle } = ViewPropTypes;
+
+class BadComponent extends Component {
```
Comment (L0wry) [inline packages/error-view/error-view.showcase.js]: could this be functional component

**Comment by L0wry** [comment]
this was a very, very exciting read

**Comment by L0wry** [comment]
also selectV2? 😢

File: packages/author-profile/author-profile.showcase.js (hunk 1 above)
Comment (craigbilner) [reply packages/author-profile/author-profile.showcase.js]: made a note :+1:

File: packages/error-view/error-view.showcase.js (hunk 2 above)
Comment (craigbilner) [reply packages/error-view/error-view.showcase.js]: made a note :+1:

**Pull request merged**